xarray/tests/test_backends.py

```diff
@@ -2712,6 +2706,14 @@ def test_attributes(self, obj) -> None:
             with pytest.raises(TypeError, match=r"Invalid attribute in Dataset.attrs."):
                 ds.to_zarr(store_target, **self.version_kwargs)
 
+    def test_vectorized_indexing_negative_step(self) -> None:
+        if not has_dask:
+            pytest.xfail(
+                reason="zarr without dask handles negative steps in slices incorrectly"
```

Comment:
Is this still failing? I fixed an obscure bug here: https://github.com/pydata/xarray/pull/7586

Reply:
It's failing in the zarr `SliceDimIndexer`:

https://github.com/zarr-developers/zarr-python/blob/6ec746ef1242dd9fec26b128cc0b3455d28ad6f0/zarr/indexing.py#L174

Reply:
Interesting. can you open an issue for this test please?